README.md

```diff
@@ -63,6 +63,7 @@ __[Official Docs](https://docs.astro.build)__
 - [Astro Static Tweet](https://github.com/rebelchris/astro-static-tweet)
 - [Astro Translation Example](https://github.com/tylergaw/astro-example-i18next)
 - [Astro Ghost CMS Starter](https://github.com/PhilDL/astro-starter-ghost) - A Ghost CMS starter for Astro
+- [Astro TAP Stack](https://github.com/codiume/the-tap-stack) - Opinionated astro starter kit (Typescript & Tailwind + Astro + Prisma & Planet scale)
```

Comment:
Hey @mohammed-elhaouari Thank you for contributing to this repo. Could you please add this to the end of the list?